tasks/copy_to_glacier/README.md

```diff
@@ -28,42 +28,91 @@ Note that this must be done for _each_ collection configured. If this list is em
+      amazonlinux:2 \
+      /bin/bash
 
-An explicit example of building the lambda package can be found in `/bin/build_tasks.sh`.
+# Install the python development binaries
+bash-4.2# yum install python3-devel
+
+# In the container cd to /data
+bash-4.2# cd /data
+
+# Go to the task
+bash-4.2# cd tasks/copy_to_glacier/
+
+# Run the tests
+bash-4.2# bin/run_tests.sh
+```
+
+Note that Bamboo will run this same script via the `bin/run_tests.sh` script found
+in the cumulus-orca base of the repo.
 
 ## Deployment
 
-Upload the zip file to AWS (either through the cli or console). Alternatively, `modules/lambdas/main.tf` shows an example of deploying this lambda through Terraform.
+The `copy_to_glacier` lambda function can be deployed using terraform using the example shown in  `modules/lambdas/main.tf` file.
 
 ```
 resource "aws_lambda_function" "copy_to_glacier" {
-  function_name    = "${var.prefix}_copy_to_glacier"
+  ## REQUIRED
+  function_name = "${var.prefix}_copy_to_glacier"
+  role          = module.restore_object_arn.restore_object_role_arn
+
+  ## OPTIONAL
+  description      = "ORCA archiving lambda used to copy data to an ORCA S3 glacier bucket."
   filename         = "${path.module}/../../tasks/copy_to_glacier/copy_to_glacier.zip"
-  source_code_hash = filemd5("${path.module}/../../tasks/copy_to_glacier/copy_to_glacier.zip")
   handler          = "copy_to_glacier.handler"
-  role             = module.restore_object_arn.restore_object_role_arn
+  memory_size      = var.orca_ingest_lambda_memory_size
   runtime          = "python3.7"
-  memory_size      = 2240
-  timeout          = 600 # 10 minutes
+  source_code_hash = filebase64sha256("${path.module}/../../tasks/copy_to_glacier/copy_to_glacier.zip")
+  tags             = local.tags
+  timeout          = var.orca_ingest_lambda_timeout
+
+  vpc_config {
+    subnet_ids         = var.lambda_subnet_ids
+    security_group_ids = [module.lambda_security_group.vpc_postgres_ingress_all_egress_id]
+  }
 
-  tags = local.default_tags
   environment {
     variables = {
-      system_bucket               = var.buckets["internal"]["name"]
-      stackName                   = var.prefix
-      CUMULUS_MESSAGE_ADAPTER_DIR = "/opt/"
+      ORCA_DEFAULT_BUCKET            = var.orca_default_bucket
+      DEFAULT_MULTIPART_CHUNKSIZE_MB = var.default_multipart_chunksize_mb
     }
   }
+}
 ```
 
 ## Input
 
-The `handler` function expects input as a Cumulus Message. The actual format of that input may change over time, so we use the `cumulus-process` package (check `requirements.txt`), which Cumulus develops and updates, to parse the input.
+The `handler` function `handler(event, context)` expects input as a Cumulus Message. Event is passed from the AWS step function workflow. The actual format of that input may change over time, so we use the [cumulus-process](https://github.com/nasa/cumulus-process-py) package (check `requirements.txt`), which Cumulus develops and updates, to parse the input.
```

Comment:
I don't think this is correct anymore, we use the cumulus-message-adapter I believe.

Reply:
yes that makes sense